Makefile

```diff
@@ -347,9 +352,9 @@ test: lint test_no_lint
 _build_container_image:
 	@[ -z "$$CONT_FILE" ] && { echo "CONT_FILE must be set"; exit 1; } || \
 	[ -z "$$TEST_IMAGE" ] && { echo "TEST_IMAGE must be set"; exit 1; }; \
-	podman image exists "$$TEST_IMAGE" && exit 0; \
+	$(_CONTAINER_TOOL) image inspect "$$TEST_IMAGE" > /dev/null 2>&1 && exit 0; \
 	echo "=========== Building container test env image ==========="; \
-	podman build -f $$CONT_FILE --tag $$TEST_IMAGE -v $$PWD:/repo:Z
+	$(_CONTAINER_TOOL) build -f $$CONT_FILE --tag $$TEST_IMAGE .
```

Comment:
@matejmatuska why is the volume replaced bz `.` ?

Reply:
The volume is no longer necessary, instead of copying using `cp` inside the containers during build, the `COPY` Dockerfile command is used. For that to work the last parameter, `context` must be set, hence the `.`.

Reply:
thanks for the clarification!